packages/mui-base/src/useAutocomplete/useAutocomplete.js

```diff
@@ -376,7 +376,11 @@ export default function useAutocomplete(props) {
     //
     // Consider this API instead once it has a better browser support:
     // .scrollIntoView({ scrollMode: 'if-needed', block: 'nearest' });
-    if (listboxNode.scrollHeight > listboxNode.clientHeight && reason !== 'mouse') {
+    if (
+      listboxNode.scrollHeight > listboxNode.clientHeight &&
+      reason !== 'mouse' &&
+      reason !== 'touch'
```

Comment:
Why was this added?

Reply:
it is an oversight, while testing. Thanks!